resources/views/livewire/stream-list.blade.php

```diff
@@ -1,5 +1,5 @@
-<div class="py-16 space-y-16">
-    @foreach ($streamsByDate as $date => $streams)
+<div class="py-16 space-y-16" id="scrollTop">
+    @foreach ((new \App\Actions\SortStreamsByDateAction())->handle($streamsByDate->getCollection()) as $date => $streams)
```

Comment:
```php
Collection::macro('groupByDate', function() {
    return (new SortStreamsByDateAction())->handle($this);
});
```
We could register a macro and just call `$streamsByDate->groupByDate()` here? 

(In addition; maybe we should rename `SortStreamsByDateAction` to `Group StreamsByDateAction`  because it's not sorting but grouping only)